canvas.py

```diff
@@ -0,0 +1,87 @@
+from flask import Flask, url_for
```

Comment:
Just ran `git checkout 8a2e4fdfaf18c614f051c871725740021f990ed0 -- canvas.py` to restore this file to how it was before being deleted.